rules/linux/privilege_escalation_env_binary.toml

```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = "Identifies Linux binary env abuse to break out from restricted environments by spawning an interactive system shell"
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via env Shell Evasion"
+references = ["https://gtfobins.github.io/gtfobins/env/"]
+risk_score = 47
+rule_id = "72d33577-f155-457d-aad3-379f9b750c97"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Privilege Escalation","GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where process.name : "env" and process.args_count == 2 and process.args : ("/bin/sh", "/bin/bash") and event.type == "start"
+'''
+
+
+[[rule.threat]]
```

Comment:
for MITRE mapping the only Technique I can see is T1059.004, privesc or autoelevation is not captured in the EQL query / rule scope.

Reply:
Corrected